### ISSUE TITLE
Provide a way to include all assets, not only chunks

### ISSUE THREAD

**Issue by laysent** [GoogleChromeLabs/preload-webpack-plugin#35]
As discussed briefly in #29, for current version of plugin, only chunks are processed. That means, for most of the cases, only JavaScript files and CSS files (which are exported using `extract-text-webpack-plugin`) can be added as `preload` using this plugin. Resources such as image and fonts, which are exported via `file-loader`, are not chunks but only assets, thus will not be handled by this plugin.

I would suggest to provide an opt-in option to include not only chunks but all assets when running the plugin. For example: 
```javascript
new PreloadWebpackPlugin({
  rel: 'preload',
  include: 'all-assets', // to include both chunks and other assets
  fileBlacklist: [/\.js$/, /\.css$/, /\.woff2$/],
});
```

**Comment by shlomiassaf** [comment]
@addyosmani I would also spice up the API.

My use case for this feature is not CSS but using the `worker-loader`.
If I choose `all-assets` or `all` I have to filter out every possible options which is not something I can maintain...

This is an `opt-out` approach, I would also like to have an `opt-in` approach... I want to get all assets but I want to filter them out, either by regex or a predicate...

**Comment by nicolas-t** [comment]
Hello,

Thank you for your work.

Any news about preloading fonts? Will the PR be merged ?

Thanks,

**Comment by beheh** [comment]
This can be closed, as #36 has landed in v2.3.0.
